task/buildah/0.1/buildah.yaml

```diff
@@ -90,6 +90,11 @@ spec:
         echo "Cannot find Dockerfile $(params.DOCKERFILE)"
         exit 1
       fi
+      if [ -n "$JVM_BUILD_WORKSPACE_ARTIFACT_CACHE_PORT_80_TCP_ADDR" ] && grep -q '^RUN \(./\)\?mvn' "$dockerfile_path"; then
```

Comment:
just wondering, where is this env var coming from?

Reply:
Kubernetes creates it out of box